src/Analyzer.Core.BuiltInRuleTests/TestTemplates/AppServicesLogs.badtemplate

```diff
@@ -0,0 +1,130 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "Location for all resources."
+            }
+        }
+    },
+    "resources": [
```

Comment:
Can you add one where some properties are true and some false?

Reply:
It's kind of hard to find the middle point between thoroughly testing an operator and testing the rule code. With our current testing system, I feel it makes sense to fully test the behavior of an operator when we add the first rule that uses it, but it wouldn't make sense to do it for all the rules. Maybe is something worth discussing? Should I create an issue to add specific end-to-end operator tests? Or to come up with some criteria about this for rule testing?

Reply:
Can you clarify what you mean @VeraBE?  I think the intent of different combinations of true/false here would be to ensure the rule is using the right operators (not to test the functionality of the operators the rule uses).  For example, if the rule were to be updated later to use different operators to try and simplify it, but the simplification missed some template scenarios, these tests should be able to detect and prevent that.

Reply:
I kind of take that comment back, I was not considering the situation of updating the rule later to use different operators. But I meant that, for example, if a rule is the first one to use the AnyOf operator I can see us wanting to test all the combinations (all the elements false, one at a time true, and all the other combinations), but I don't see us doing that for every rule, that could potentially have 10 elements in the AnyOf array